packages/neo-one-smart-contract-lib/src/__tests__/__snapshots__/RedToken.test.ts.snap

```diff
@@ -2,4 +2,4 @@
 
 exports[`RedToken properties + issue + balanceOf + totalSupply + transfer: transfer consume 1`] = `"0"`;
 
-exports[`RedToken properties + issue + balanceOf + totalSupply + transfer: transfer cost 1`] = `"3.389"`;
```

Comment:
Curious: does your contract affect the RedToken test? Odd that it makes it cost more gas, no?

Reply:
Shouldn't be.  I'll look into this.  Did any Opcode changes go in?

Reply:
This happens because the compiler is grabbing everything that's being exported by `@neo-one/smart-contracts-lib` -- Alex indicated there's a plan to clean this up.